ppr-ui/src/components/mhrRegistration/ReviewConfirm/YourHomeReview.vue

```diff
@@ -199,14 +191,15 @@ export default defineComponent({
       }),
       engineerDisplayDate: computed((): string => {
         return yyyyMmDdToPacificDate(getMhrRegistrationHomeDescription.value?.engineerDate, true)
-      })
+      }),
+      showOtherInformation: computed(() => !isMhrManufacturerRegistration.value),
+      showRebuiltStatus: computed(() => !isMhrManufacturerRegistration.value),
```

Comment:
I don't mind this but i'm not sure i see the purpose.
You can return the getters from the setup function and use them directly in the template. 

I understand the getter names are pretty explicit and sometimes longer but you also don't need to the `.value` ref deconstruction in the template either.

Your call but i feel you should handle them in the same manner as the props. I saw earlier in the pr the refactoring to remove the local references (ie the props defined/assigned locally -> refactored to use directly), feels like the same scenario here.

If you need to mutate a prop or have more complex computed values utilizing a getter, i totally understand but not sure i see the case for `false` scenario of a boolean.

Your call!

Reply:
I agree, the idea behind it originally was that if we have more flows that might show or not show the section, we can have more control, or if they decided to add one section but the other in the future, it can be easily changed. 

However, I do think its clearer to just put in a template, and if we need to make changes we can do it when we hit that point.